### PR TITLE
Remove margins left/right from header

### LINKED ISSUE #1016: Menu button is clued to the network switcher in a mobile view
Related to #1011 , #953

1. Open the Explorer app in a mobile view
2. Verify the Menu button

**AR**: Menu button is clued to the network switcher
clued

**ER**: the button is not clued

### PR DESCRIPTION
**Pull request by henrypalacios**
# Summary

Closes #1016 

## To test
Go to https://pr1026--gpui.review.gnosisdev.com and check the **header** in mobile 320px resolution view

### CODE REVIEW

**Comment by alfetopito** [comment]
Doesn't really fix what seems to be the reported issue
Screen Shot 2022-02-02 at 13 52 23

**Comment by henrypalacios** [comment]
@alfetopito It seems that there are very small resolutions :face_in_clouds: 

What do you think if we move the network selector to the next line with small resolutions?
Selection_447

**Comment by elena-zh** [comment]
Hey @henrypalacios , could we do something like this (adjust these properties) in order to resolve this issue:  https://watch.screencastify.com/v/rdNeImPu8KIItBDFGQSH ?

**Comment by henrypalacios** [comment]
Applying @elena-zh 's suggestion and reducing the image by 2% we can get this :point_down: 
Selection_449

Let me know whay you think.

**Review by elena-zh — APPROVED** [review]
Looks great to me!

File: src/apps/explorer/styled.ts [hunk 1]
```diff
@@ -46,9 +46,14 @@ export const MainWrapper = styled.div`
...
+
     header {
-      margin-left: auto;
-      margin-right: auto;
+      img {
+        max-width: 98%;
+      }
+      div.network-selector-container {
+        padding-right: 2rem;
+      }
```
Comment (alfetopito) [inline src/apps/explorer/styled.ts:56]: Wouldn't make sense instead to set that property onto the `SelectorContainer` component instead?

**Review by alfetopito — COMMENTED** [review]
It's working great now!

Just need to figure out the CSS part

**Review by alfetopito — APPROVED** [review]
Looks good now 👍 

I've asked @biocom to review it as well BTW

No need to wait though, if there are any comments you can address in a follow up PR

File: src/apps/explorer/styled.ts (hunk 1 above)
Comment (henrypalacios) [reply src/apps/explorer/styled.ts]: Yes, I have taken the responsibility for the style to each component.

**Review by biocom — APPROVED** [review]

**Pull request merged**